### PR TITLE
Translation of Resource Hints chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of [#586](https://github.com/HTTPArchive/almanac.httparchive.org/issues/586)

### CODE REVIEW

**Comment by ksakae1216** [comment]
@tunetheweb Test Website failed.
Same error as below. Do you know how to fix??
https://github.com/HTTPArchive/almanac.httparchive.org/pull/3023

**Comment by tunetheweb** [comment]
Looks like another Werkzeug bug. Have pinned it and seems to pass now. Will do a review of this later as heading out now.

**Comment by tunetheweb** [comment]
Raised this btw: https://github.com/pallets/werkzeug/issues/2481

File: src/content/ja/2021/resource-hints.md [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+<figcaption>{{ figure_link(caption="モバイル端末でリソースヒントを使用しているもっとも普及しているサードパーティ接続。", sheets_gid="1254656281", sql_file="preload_host_by_url.sql") }}</figcaption>
+</figure>
+
+上の表を分析すると、`preload` ヒントを含む全ページの36.7%がadservice.google.comでホストされているリソースをプリロードしています。s.w.orgホストは `dns-prefetch` でもっとも普及しているドメインで、WordPressサイト（バージョン4.6以降）で、ブラウザがネイティブの絵文字をサポートしていないと検出された場合にTwemoji CDNからのSVG画像を読み込むために使用されています。Google Fonts関連のサービスでは、`fonts.gstatic.com` と `fonts.googleapis.com` が `preconnect` ディレクティブを使用するホストとしてもっとも普及しているようです。
+
+{{ figure_markup(
+  image="google-fonts.png",
+  alt="Google Fontsのフォントを埋め込むための手順",
+  caption='Google Fontsの説明では、fonts.gstatic.comとfonts.googleapis.comへ事前接続するようになっています。（出典: <a hreflang="en" href="https://fonts.google.com/">Google Fonts</a>）',
+  description='Code snippet titled "Use on the web" and subtitled with "To embed a font, copy the code into the `<head>` of your html". Code snippet states `<link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">`',
```
Comment (tunetheweb) [inline src/content/ja/2021/resource-hints.md:561]: Should we translate bits of this description?

**Review by tunetheweb — CHANGES_REQUESTED** [review]

File: src/content/ja/2021/resource-hints.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2021/resource-hints.md]: Thanks, I translated

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**